egs/librispeech/s5/local/rnnlm/tuning/run_tdnn_lstm_1a.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# Copyright 2012  Johns Hopkins University (author: Daniel Povey)
+#           2018  Ke Li
+
+# This script trains LMs on the librispeech 960 hours training data.
+
+# rnnlm/train_rnnlm.sh: best iteration (out of 26) was 21, linking it to final iteration.
+# rnnlm/train_rnnlm.sh: train/dev perplexity was 118.4 / 152.6.
+# Train objf: -5.74 -5.51 -5.38 -5.29 -5.22 -5.16 -5.12 -5.08 -5.05 -5.02 -4.99 -4.97 -4.97 -4.93 -4.90 -4.87 -4.84 -4.82 -4.79 -4.77 -4.75 -4.73 -4.71 -4.69 -4.67
+# Dev objf:   -6.00 -5.61 -5.45 -5.36 -5.29 -5.24 -5.20 -5.18 -5.16 -5.13 -5.12 -5.11 -5.11 -5.09 -5.07 -5.06 -5.05 -5.04 -5.03 -5.03 -5.03 -5.03 -5.03 -5.03 -5.03 -5.03
+
```

Comment:
Can you please include some WER comparisons?

Reply:
sure, I have WER numbers on an old chain model. I'm running a new one now. will update the rescoring results on that in a few days.